### PR TITLE
annotations fixed

### PR DESCRIPTION
**Pull request by satanu22ou**
Removed tags and fixed other issues

### CODE REVIEW

**Comment by cegme** [comment]
Pull the latest and re-request a review

**Review by satanu22ou — COMMENTED** [review]
I have updated the pull and resubmitted

**Comment by cegme** [comment]
@satanu22ou Please pull the latest (resolve conficts)

**Review by satanu22ou — COMMENTED** [review]
Please check

**Comment by cegme** [comment]
You need to pull the latest and fix conflicts 
image

File: unredactor.tsv [hunk 1]
```diff
@@ -1892,3 +1892,93 @@ josudha10	validation	Newhart	Every thing had been in the mind of an autistic chi
...
+satanu22ou	training	Leandro Lucchetti	Un Gatto nel Cervello/Cat in the Brain is one of the goriest horror movies ever made.There is a lot of blood and gore,including chainsaw butchery,bloody stabbings and numerous decapitations.The film is also interesting as self parody of Fulci,but the gore and violence is the key element in it.Some of the gore FX were taken from own Fulci's movies Quando Alice Ruppe lo Specchio and I Fantasmi di Sodoma(both 1988),plus gore FX taken from Fulci-supervised The Snake House aka Bloody Psycho by █████████████████,Massacre by Andrea Bianchi,Non Avere Paura Della Zia Marta by Mario Bianchi,Non Si Sevizia i Bambini by Giovanni Simonelli and Luna di Sangue aka Fuga dalla Morte by Enzo Milioni(all 1989).The scene where Brett Halsey beats the woman's face to pulp is from Quando Alice Ruppe lo Specchio,a film Fulci had made for Italian TV in 1988.The chainsawing of the female corpse at the beginning is taken from the same film,as is the head in the microwave and the guy that gets driven over and over again.Highly recommended,especially if you like extreme cinema!
```
Comment (cegme) [inline unredactor.tsv:1898]: I will approve but please check these rows

**Review by cegme — APPROVED** [review]
Please double check.

**Pull request merged**